### ISSUE TITLE
Indicate when a comment has been deleted

### ISSUE THREAD

**Issue by Cleop** [dwyl/github-backup#58]
As a gh-backup user,
I want to know when a comment has been deleted 
So that I can see all the activity happening on github reflected on the backup app.

- [ ] Catch webhook from Github for a deleted comment
- [ ] Update the comment table with a new column to indicate if a comment is deleted (`deleted: true` or `false`). Default of `false`.
- [ ] Update the `deleted` column when a comment is deleted to `true`
- [ ] Update the UI when a comment is deleted to reflect the gh UI:
![image](https://user-images.githubusercontent.com/16775804/37088515-8aa0405c-21f5-11e8-9762-939cac1c0475.png)

**Comment by Cleop** [comment]
**Question - if a comment is deleted and we add the GH UI for this, what colour should the comment box be for the last active comment? Should it be white as the latest comment normally is or as the latest activity on the comment is that it's been deleted should all the records of the comment now have a grey background?**

![image](https://user-images.githubusercontent.com/16775804/37092237-f383735c-2202-11e8-9247-efa39a6ded28.png)

**Comment by Cleop** [comment]
We need to rethink our schema for doing this. We can't just add a `deleted` column because we need to also record the `author` of the deleting action. So far we only record an `author` value when we add a new version. 

So instead given that there will be lots of github events to record in the future e.g. adding a label, putting an issue in a milestone etc. We think it would be better to have an `events` table with actions like - deleted comment, adds/removes label, adds/ removes from milestone etc.

**Comment by SimonLab** [comment]
I think the only event linked to the comment will be `deleted`. The other events (new label, isssue assigned,...) are linked to the issue not the comment itself

**Comment by SimonLab** [comment]
A solution to save the user who deleted a comment is to create another new field in the comment table `deleted_by` which will be set to null by default.
When a comment is deleted we can then update this field with the name of the user and later one when #46 is done we can create a reference to the user table

**Comment by Cleop** [comment]
This is the UI for now:
![image](https://user-images.githubusercontent.com/16775804/37155281-69c34128-22da-11e8-849d-a23d3847c881.png)

**Comment by nelsonic** [comment]
@Cleop copy is good. 
Could we use https://fontawesome.com/icons/trash?style=solid ?

**Comment by Cleop** [comment]
@nelsonic - thanks for the font awesome asset suggestion. Shall I create this as a separate issue or complete it as part of this one?

**Comment by nelsonic** [comment]
@Cleop you _can_ create a new issue (_linked back to this one_).
(_I kinda **prefer** that to changing the acceptance criteria in `this` issue ... don't you?_)

**Comment by Cleop** [comment]
I agree, just wanted to check as you hadn't closed this issue (and it was in `please-test`). I'll do that now 👍
